carmin-server/server/properties_validation.py

```diff
@@ -0,0 +1,52 @@
+"""Properties Validation performs validation on the `server-config.json` file,
+which contains a description of the PlatformProperties object.
+"""
+import os
+import json
+from .resources.models.platform_properties import PlatformPropertiesSchema
+from .common.exceptions import MissingRequiredParameterError
+
+
+def properties_validation(config_data=None):
+    """Performs validation on server_config.json file. Checks for required
+    parameters and supported options."""
+    supported_protocols = [
```

Comment:
I would specify these information in a private config file instead of in the function on its own.